10up/headstartwp
# Category Path Matching

### Description of the Change



Closes #369 

### How to test the Change
1. `npm run dev`
2. Open an category (or tag) archive
3. Make sure it loads archive pages correctly.
4. Then add a fake parent term e.g: `/category/fake-parent/real-term-name` and make sure it 404.
5. Do the same for tags archive
6. In `headstartwp.config.js`, change `matchArchivePath` to false, now confirm step 4 does not issue a 404!

### Checklist:
